src/views/Home.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <b-container class="p-0">
+    <b-row>
+      <b-col lg="8" cols="12">
+        <div v-if="!userIsAuthenticated" class="jumbotron noauth">
+          <h1>Observation Portal</h1>
+          <p>This is your access point to Las Cumbres Observatory's global network of telescopes. Registered users can:</p>
+          <ul class="list-unstyled">
+            <li>
+              <i class="fa fa-users fa-3x fa-pull-left" aria-hidden="true" />
+              <span>Submit proposals and manage the membership of their research teams.</span>
```

Comment:
Not submit proposals in the example... Maybe change the test to say this is an Example Observation Portal, users can blah blah blah